solutions/advanced/cses-1694.mdx

```diff
@@ -2,80 +2,87 @@
 id: cses-1694
 source: CSES
 title: Download Speed
-author: Dong Liu
+author: Ahmet Ala
```

Comment:
bro really removed the og author

Reply:
This is an internal Solution page and the only thing on the page is the code, which I have changed. Should I still keep the previous author's name?

Reply:
yeah i don't think anyone has ever deleted previous authors here

Reply:
alright added back his name, I should have kept his name since he is the one who created the 'internal solution page for download speed'.